pkg/frontend/load.go

```diff
@@ -1248,6 +1256,7 @@ when force is true, batchsize will be changed.
 */
 func writeBatchToStorage(handler *WriteBatchHandler, force bool) error {
 	var err error = nil
+	handler.threadCnt = 1
```

Comment:
1. Update the shared variable with sync.atomic  in the concurrent environment.
2. Put the access to the threadCnt just before the tableHandler.Write.